bundler/lib/bundler/man/bundle-config.1

```diff
@@ -220,6 +224,9 @@ The following is a list of all configuration keys and their purpose\. You can le
 \fBno_prune\fR (\fBBUNDLE_NO_PRUNE\fR): Whether Bundler should leave outdated gems unpruned when caching\.
 .
 .IP "\(bu" 4
+\fBonly\fR (\fBBUNDLE_ONLY\fR): A space\-separated list of groups to install only gems of the specified groups\.
```

Comment:
Is it intentional this is space-separated instead of `:`-separated like BUNDLE_WITH/WITHOUT?
Also the other comments on this PR assume it would be `:`-separated so that might be better/more intuitive?

Reply:
Maybe this is a documentation issue. Implementation and tests here seem to use `:`.